src/mock/parsec/tests.rs

```diff
@@ -226,6 +226,75 @@ fn reevaluate_previously_insufficient_votes() {
     assert_consensus_on_remove(&mut nodes, dave);
 }
 
+#[test]
+fn newly_joined_node_does_not_cause_premature_consensus() {
+    init_mock();
+
+    // Create the genesis group.
+    let mut genesis_group = BTreeSet::new();
```

Comment:
Sure, I can do that. Ideally I would use this: https://docs.rs/maplit/1.0.2/maplit/, but not sure it's worth bringing another dependency for it (although it would be only dev-dependency).

Reply:
Maybe not yet bring the dependency, we can always do it later, should be easier to refactor from the suggested pattern. (in parsec we also had a btreeset! macro I think, but lets keep it simple for now.)